doc/Programs/03-environment-variables.pod6

## Problem or new feature

Newly added file is not test clean - it's failing the valid POD test, which may be causing the other failures.

## Suggestions

```
$ cat retest
#!/usr/bin/env raku
%*ENV<TEST_FILES>='META6.json doc/Programs/03-environment-variables.pod6';
say %*ENV<TEST_FILES>;
run(<make xtest>).so
$ ./retest
META6.json doc/Programs/03-environment-variables.pod6
if [ "3" != "" ]; then prove -j 3 -e raku t xt; else prove -e raku t xt; fi
t/02-tests-valid.t ......... skipped: No test files specified
t/00-meta.t ................ ok
===(       2;2  1/1  0/?  0/? )=========================================# Failed test 'doc/Programs/03-environment-variables.pod6 has clean Pod6'
# at t/02-pod-valid.t line 51
# You failed 1 test of 1
t/02-pod-valid.t ........... Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests
t/03-pod6.t ................ ok
t/05-pod-convenience.t ..... ok
t/07-tabs.t ................ ok
xt/aspell.t ................ ok
xt/braces.t ................ ok
xt/check-signatures.t ...... skipped: To run check-signatures, please specify the path to the Rakudo git checkout with the RAKUDO_SRC environment variable
===(      18;10  0/?  0/? )================================================SORRY!=== Error while compiling /Volumes/BUKKIT/Users/coke/sandbox/raku-doc/doc/Programs/03-environment-variables.pod6
Preceding context expects a term, but found infix = instead.
Did you make a mistake in Pod syntax?
at /Volumes/BUKKIT/Users/coke/sandbox/raku-doc/doc/Programs/03-environment-variables.pod6:262
------> <BOL>⏏<EOL>

  in sub extract-pod at /Volumes/BUKKIT/Users/coke/sandbox/raku-doc/lib/Pod/Convenience.pm6 (Pod::Convenience) line 127
  in block <unit> at xt/camelia-invocations.t line 37

xt/camelia-invocations.t ... Dubious, test returned 1 (wstat 256, 0x100)
No subtests run
xt/double-dots.t ........... ok
xt/duplicates.t ............ ok
xt/final-newline.t ......... ok
===(      22;14  0/?  0/? )================================================SORRY!=== Error while compiling /Volumes/BUKKIT/Users/coke/sandbox/raku-doc/doc/Programs/03-environment-variables.pod6
Preceding context expects a term, but found infix = instead.
Did you make a mistake in Pod syntax?
at /Volumes/BUKKIT/Users/coke/sandbox/raku-doc/doc/Programs/03-environment-variables.pod6:262
------> <BOL>⏏<EOL>

  in sub extract-pod at /Volumes/BUKKIT/Users/coke/sandbox/raku-doc/lib/Pod/Convenience.pm6 (Pod::Convenience) line 127
  in sub code-blocks at xt/examples-compilation.t line 135
  in block  at xt/examples-compilation.t line 38
  in block <unit> at xt/examples-compilation.t line 36

xt/examples-compilation.t .. Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests
===(      23;16  1/1  0/?  0/? )========================================# Failed test 'doc/Programs/03-environment-variables.pod6 has inconsistent capitalised headings on lines: 5
# # =SUBTITLE How to run Rakudo, a Raku implementation, and modify its behavior'
# at xt/headings.t line 70
# You failed 1 test of 1
xt/headings.t .............. Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests
xt/links-not-links.t ....... skipped: (no reason given)
xt/perl-nbsp.t ............. ok
xt/return-type.t ........... skipped: (no reason given)
xt/trailing-whitespace.t ... ok
xt/space-after-comma.t ..... ok
===(      27;22  0/1  0/? )================================================SORRY!=== Error while compiling /Volumes/BUKKIT/Users/coke/sandbox/raku-doc/doc/Programs/03-environment-variables.pod6
Preceding context expects a term, but found infix = instead.
Did you make a mistake in Pod syntax?
at /Volumes/BUKKIT/Users/coke/sandbox/raku-doc/doc/Programs/03-environment-variables.pod6:262
------> <BOL>⏏<EOL>

  in sub extract-pod at /Volumes/BUKKIT/Users/coke/sandbox/raku-doc/lib/Pod/Convenience.pm6 (Pod::Convenience) line 127
  in block  at xt/routine-definitions.t line 23
  in block <unit> at xt/routine-definitions.t line 21

xt/routine-definitions.t ... Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests
xt/words.t ................. ok
xt/type-graph.t ............ ok
xt/word-variants.t ......... ok

Test Summary Report
-------------------
t/02-pod-valid.t         (Wstat: 256 Tests: 1 Failed: 1)
  Failed test:  1
  Non-zero exit status: 1
xt/camelia-invocations.t (Wstat: 256 Tests: 0 Failed: 0)
  Non-zero exit status: 1
  Parse errors: No plan found in TAP output
xt/examples-compilation.t (Wstat: 256 Tests: 0 Failed: 0)
  Non-zero exit status: 1
  Parse errors: Bad plan.  You planned 1 tests but ran 0.
xt/headings.t            (Wstat: 256 Tests: 1 Failed: 1)
  Failed test:  1
  Non-zero exit status: 1
xt/routine-definitions.t (Wstat: 256 Tests: 0 Failed: 0)
  Non-zero exit status: 1
  Parse errors: Bad plan.  You planned 1 tests but ran 0.
Files=24, Tests=361, 28 wallclock secs ( 0.16 usr  0.06 sys + 79.13 cusr  7.11 csys = 86.46 CPU)
Result: FAIL
make: *** [xtest] Error 1
```

--- a/doc/Programs/03-environment-variables.pod6
+++ b/doc/Programs/03-environment-variables.pod6
@@ -11,7 +11,8 @@ raku - Rakudo Raku Compiler
 
 =head1 SYNOPSIS
 
- raku [switches] [--] [programfile] [arguments]
+=for code :lang<usage>
+raku [switches] [--] [programfile] [arguments]
 
 =head1 DESCRIPTION
 
@@ -19,42 +20,46 @@ With no arguments, enters a REPL. With a C<[programfile]> or the C<-e>
 option, compiles the given program and by default also executes the
 compiled code.
 
-  -c                   check syntax only (runs BEGIN and CHECK blocks)
-  --doc                extract documentation and print it as text
-  -e program           one line of program, strict is enabled by default
-  -h, --help           display this help text
-  -n                   run program once for each line of input
-  -p                   same as -n, but also print $_ at the end of lines
-  -I path              adds the path to the module search path
-  -M module            loads the module prior to running the program
-  --target=[stage]     specify compilation stage to emit
-  --optimize=[level]   use the given level of optimization (0..3)
-  --encoding=[mode]    specify string encoding mode
-  -o, --output=[name]  specify name of output file
-  -v, --version        display version information
-  --stagestats         display time spent in the compilation stages
-  --ll-exception       display a low level backtrace on errors
-  --profile            write profile information as HTML file (MoarVM)
-  --profile-filename   provide a different filename (also allows .json)
-  --doc=[module]       Use Pod::To::[module] to render inline documentation.
-  --full-cleanup       try to free all memory and exit cleanly (MoarVM)
-  --debug-port=port    listen for incoming debugger connections (MoarVM)
-  --debug-suspend      pause execution at the entry point (MoarVM)
-  --tracing            output a line to stderr on every interpreter instr (only
-                       if enabled in MoarVM)
+=begin code :lang<usage>
+-c                   check syntax only (runs BEGIN and CHECK blocks)
+--doc                extract documentation and print it as text
+-e program           one line of program, strict is enabled by default
+-h, --help           display this help text
+-n                   run program once for each line of input
+-p                   same as -n, but also print $_ at the end of lines
+-I path              adds the path to the module search path
+-M module            loads the module prior to running the program
+--target=[stage]     specify compilation stage to emit
+--optimize=[level]   use the given level of optimization (0..3)
+--encoding=[mode]    specify string encoding mode
+-o, --output=[name]  specify name of output file
+-v, --version        display version information
+--stagestats         display time spent in the compilation stages
+--ll-exception       display a low level backtrace on errors
+--profile            write profile information as HTML file (MoarVM)
+--profile-filename   provide a different filename (also allows .json)
+--doc=[module]       Use Pod::To::[module] to render inline documentation.
+--full-cleanup       try to free all memory and exit cleanly (MoarVM)
+--debug-port=port    listen for incoming debugger connections (MoarVM)
+--debug-suspend      pause execution at the entry point (MoarVM)
+--tracing            output a line to stderr on every interpreter instr (only
+                     if enabled in MoarVM)
+=end code
 
 Note that only boolean single-letter options may be bundled.
 
 The supported values for C<--target> are:
 
-  Target     Backend  Description
-  ======     =======  ===========
-  parse      all      a representation of the parse tree
-  ast        all      an abstract syntax tree (before optimizations)
-  optimize   all      an abstract syntax tree (after optimizations)
+=begin code :lang<text>
+Target     Backend  Description
+======     =======  ===========
+parse      all      a representation of the parse tree
+ast        all      an abstract syntax tree (before optimizations)
+optimize   all      an abstract syntax tree (after optimizations)
 
-  mbc        MoarVM   MoarVM byte code
-  jar        JVM      JVM archive
+mbc        MoarVM   MoarVM byte code
+jar        JVM      JVM archive
+=end code
 
 For C<--profile-filename>, specifying a name ending in C<.json> will write a raw
 JSON profile dump. The default if this is omitted is C<profile-I<[timestamp]>.html>.
@@ -249,7 +254,8 @@ everything that is printed to C<STDOUT> and C<STDERR>.
 
 To receive the output of the program it suffices to redirect it externally:
 
-  rakuw.exe script.raku >stdout.txt 2>stderr.txt
+=for code :lang<batch>
+rakuw.exe script.raku >stdout.txt 2>stderr.txt
 
 =head1 AUTHORS
 
@@ -257,5 +263,7 @@ Written by the Rakudo contributors, see the CREDITS file.
 
 This manual page was written by Reini Urban, Moritz Lenz and the Rakudo
 contributors.
+
+=end pod
 
 # vim: expandtab softtabstop=4 shiftwidth=4 ft=perl6

--- a/xt/headings.t
+++ b/xt/headings.t
@@ -26,7 +26,7 @@ for @files -> $file {
             # proper names, macros, acronyms, and other exceptions
             $title ~~ s:g/ <|w> (
                 I
-                | Raku | Perl 6 | Pod6 | P6 | C3 | NQP
+                | Raku | Rakudo | Perl 6 | Pod6 | P6 | C3 | NQP
                 | AST | EVAL | PRE | POST | CLI | MOP
                 | TITLE | SUBTITLE | "MONKEY-TYPING"
                 | API | TCP | UDP | FAQ

--- a/xt/pws/words.pws
+++ b/xt/pws/words.pws
@@ -410,6 +410,7 @@ evalbot
 evalbytes
 evaled
 evalfile
+exe
 execl
 executables
 execv
@@ -801,6 +802,7 @@ moarperf
 moarvm
 moarvm's
 modulehow
+moduleloader
 mojo
 mojolicious
 moritz
@@ -1121,9 +1123,11 @@ rakudo's
 rakudobrew
 rakudoc
 rakudolib
+rakudow
 rakulib
 rakumod
 rakutest
+rakuw
 ratstr
 readchars
 readdir
@@ -1189,6 +1193,7 @@ rubygems
 rubylib
 runge
 rungekutta
+runtime
 runtime's
 rvalue
 rvalues

